test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -236,4 +236,4 @@ intervals:
   default/wait-control-plane-upgrade: ["35m", "30s"]
   default/wait-addon-status: ["10m", "30s"]
   default/wait-create-identity: ["1m", "10s"]
-  default/wait-create-service: ["60s", "10s"]
+  default/wait-create-service: ["5m", "10s"]
```

Comment:
Do we need this?

Reply:
yep, both the eks test e2e and the unmanaged e2e test use this interval. i increased the timeout here to see if that would help fix any of the previous e2e eks errors i was getting, but it looks like it did not. 

if you want, we can change back to 60s